src/std_illuminants.rs

```diff
@@ -51,9 +38,18 @@ A static reference to the spectra can be obtained using the "spectrum" method.
         println!{"{spc}"};
     }
 ```
+*/
```

Comment:
It was slightly unclear to me what the original intention here was. The format was:

```
<doc comments>
<import statements>
<more doc comments>
```
I moved the imports to after the doc comments and merged the comments into one block. Was this how it was supposed to be? :)